### DISCUSSION TITLE
Snap package is ready and well maintained

### DISCUSSION THREAD

**Discussion by soumyaDghosh** [dialect-app/dialect#327] [Show and tell]
Hello, I have packaged dialect as a snap, and it works pretty well. If you like to see it and want it to add to this organization, we can have it. And also I can transfer the ownership to this organization account in snap store also. Thanks.

**[Dialect Snap](https://snapcraft.io/dialect)**

**Comment by mufeedali** [comment]
Hi,

Very cool to see a snap package happening! Thanks for the work! Much appreciated.

I think it's best if it stays under your ownership, i.e. outside the organization, as it is a third party package. As long as there are no modifications or patches to the source code, we can add it to the README as an installation method.

Thanks :smile:

**Reply by soumyaDghosh** [reply]
[![Get it from the Snap Store] (https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/dialect)

**Comment by soumyaDghosh** [comment]
Thank you. I'll share a badge via a PR then. I have also packaged wordbook and share-preview. They haven''t got listed yet, waiting in the review queue. Dialect currently has 200 active devices.
